src/main/kotlin/io/snyk/plugin/analytics/ItlyHelper.kt

```diff
@@ -0,0 +1,46 @@
+package io.snyk.plugin.analytics
+
+import ai.deepcode.javaclient.core.SuggestionForFile
+import io.snyk.plugin.services.SnykApplicationSettingsStateService
+import snyk.analytics.AnalysisIsReady.AnalysisType
+import snyk.analytics.IssueIsViewed
+import snyk.analytics.ItlyOverrideHelper
+
+object ItlyHelper {
```

Comment:
top level funs?

Reply:
It sounds reasonable in this case, done.